src/test/java/com/google/sps/data/PotentialMatchAlgorithmTest.java

```diff
@@ -0,0 +1,185 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import java.util.Set;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import com.google.common.collect.ImmutableSet;
+import com.google.common.collect.ImmutableMap;
+import com.google.sps.data.friend_map.UserFriendsMap;
+import com.google.sps.data.friend_map.UserNode;
+import com.google.sps.data.PotentialMatchAlgorithm;
+
+/* NOTE: PM = Potential Match .*/
+@RunWith(JUnit4.class)
+public class PotentialMatchAlgorithmTest {
+
+  private static final String USER_A_ID = "12345";
+  private static final String USER_B_ID = "23456";
+  private static final String USER_C_ID = "34567";
+  private static final String USER_D_ID = "45678";
+  private static final String USER_E_ID = "56789";
+
+  /**
+  * Tests if correct potential matches are found for a single user with zero friends
+  *
+  * <p>Should result in a map with a single key that maps to an empty set.
+  */
+  @Test
+  public void oneUserNoFriendsPMTest() {
+    UserNode userA = new UserNode(USER_A_ID, ImmutableSet.of());
+
+    Set<UserNode> oneUserNodeSet = ImmutableSet.of(userA);
+    UserFriendsMap resultingFriendsMap = new UserFriendsMap(oneUserNodeSet);
+    ImmutableMap<String, ImmutableSet<String>> resultingPotentialMatches = 
```

Comment:
`resultingPotentialMatches` -> `potentialMatches`

Reply:
Got it!